src/components/10-atoms/link/react/story.jsx

```diff
@@ -34,6 +34,22 @@ storiesOf('Atoms/Link/React', module)
     );
   })
 
+  .add('Link - icon onClick', () => {
+    return wrap(
+      <AXALinkReact
+        onClick={ev => {
+          ev.preventDefault();
```

Comment:
I somehow don't like that this preventDefault is needed. That should be automatically prevented, if an onClick handler is passed down. We do that for other components too. Now we have kind of an inconsistency in how we do that.

Reply:
done